core/client/app/templates/setup/three.hbs

```diff
@@ -12,7 +12,8 @@
 sally.sanders@example.com" value=users}}
     </form>
 
-    <button {{action "signin"}} class="btn btn-default btn-lg btn-block {{buttonClass}}">
+    <button {{action 'invite'}} class="btn btn-default btn-lg btn-block {{buttonClass}}">
         {{buttonText}}
     </button>
+    <a {{action 'signin'}} href="#" class="gh-flow-skip">I'll do this later, take me to my blog!</a> 
```

Comment:
This can be switched to a simple link-to instead of a button with an action:

```handlebars
{{#link-to "posts" class="gh-flow-skip"}}
    I'll do this later, take me to my blog!
{{/link-to}}
```